install/operator/pkg/syndesis/template/install.go

```diff
@@ -44,7 +46,25 @@ func GetInstallResourcesAsRuntimeObjects(scheme *runtime.Scheme, syndesis *v1alp
 }
 
 func GetInstallResources(scheme *runtime.Scheme, syndesis *v1alpha1.Syndesis, params InstallParams) ([]runtime.RawExtension, error) {
-	res, err := util.LoadResourceFromFile(scheme, *configuration.TemplateLocation)
+	var templateLocation string
+
+	if releaseVersion := *configuration.ReleaseVersion; len(releaseVersion) > 0 {
+		var fileUrl bytes.Buffer
```

Comment:
I'd go with `fmt.Sprintf` here instead, something like:
```suggestion
		fileUrl := fmt.Sprintf("https://raw.githubusercontent.com/syndesisio/fuse-online-install/%s/resources/fuse-online-template.yml", releaseVersion)
```

Would also be good to put `"https://raw.githubusercontent.com/syndesisio/fuse-online-install/%s/resources/fuse-online-template.yml"` into a constant.